src/types/index.ts

```diff
@@ -170,6 +170,7 @@ export interface IAccountOverview extends Partial<Omit<IAccount, 'address'>> {
   contractCreate?: boolean;
   aens?: boolean;
   label?: TranslateResult;
+  wallet?: string;
```

Comment:
Can you add a description for this property? Maybe some possible values also?

Reply:
Added